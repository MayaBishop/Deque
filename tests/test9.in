#testing front and back commands with a deque of 1 and an empty deque
enqueue_front 1
front 1
back 1
clear
front 1
back 1
clear
