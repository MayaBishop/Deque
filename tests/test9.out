success
success
success
success
failure
failure
success
